.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
-  - "9.5"
```

Comment:
What will Travis do now node-js-version-wise? Use the latest stable?

Reply:
That's what I thought. I had added this as a workaround until we fixed the issue, but now we seem to be getting node "0.10". So I'll set this to just "9" to get the latest 9.* hopefully.